### ISSUE TITLE
rp_ignore_errors doesn't ignore connection issues

### ISSUE THREAD

**Issue by akropyva-apixio** [reportportal/agent-python-pytest#128]
For example reporpotral server is down and we have rp_ignore_errors = True and rp_verify_ssl = False into pytest.ini.
We have an error like this:
```
INTERNALERROR>   File "/home/akropyva/Projects/apx-qa-tests/venv/lib/python3.7/site-packages/pytest_reportportal/plugin.py", line 57, in pytest_sessionstart
INTERNALERROR>     verify_ssl=session.config.getini('rp_verify_ssl')
INTERNALERROR>   File "/home/akropyva/Projects/apx-qa-tests/venv/lib/python3.7/site-packages/pytest_reportportal/service.py", line 98, in init_service
INTERNALERROR>     self.project_settiings = self.RP.rp_client.get_project_settings() if self.RP else None
INTERNALERROR>   File "/home/akropyva/Projects/apx-qa-tests/venv/src/reportportal-client/reportportal_client/service.py", line 230, in get_project_settings
INTERNALERROR>     r = self.session.get(url=url, json={}, verify=self.verify_ssl)
INTERNALERROR>   File "/home/akropyva/Projects/apx-qa-tests/venv/lib/python3.7/site-packages/requests/sessions.py", line 546, in get
INTERNALERROR>     return self.request('GET', url, **kwargs)
INTERNALERROR>   File "/home/akropyva/Projects/apx-qa-tests/venv/lib/python3.7/site-packages/requests/sessions.py", line 533, in request
INTERNALERROR>     resp = self.send(prep, **send_kwargs)
INTERNALERROR>   File "/home/akropyva/Projects/apx-qa-tests/venv/lib/python3.7/site-packages/requests/sessions.py", line 646, in send
INTERNALERROR>     r = adapter.send(request, **kwargs)
INTERNALERROR>   File "/home/akropyva/Projects/apx-qa-tests/venv/lib/python3.7/site-packages/requests/adapters.py", line 516, in send
INTERNALERROR>     raise ConnectionError(e, request=request)
INTERNALERROR> requests.exceptions.ConnectionError: HTTPSConnectionPool(host='reportposrtal.apixio.com', port=443): Max retries exceeded with url: /api/v1/vkushnarenko-apixio_personal/settings (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6bd34ab748>: Failed to establish a new connection: [Errno -2] Name or service not known'))
```
init_service => get_project_settings => r = self.session.get(url=url, json={}, verify=self.verify_ssl) which causes an error past parameter rp_ignore_errors

**Comment by iivanou** [comment]
The main cause of the issue was fixed in https://github.com/reportportal/agent-python-pytest/pull/127.

In addition to that fix, we will need to make changes in code where the RPLogHandler is being setup.

**Comment by iivanou** [comment]
Docs have been updated. A fix is available in 1.0.8 release.
